baby-a3c.py

```diff
@@ -69,13 +69,13 @@ def __init__(self, params, lr=1e-3, betas=(0.9, 0.999), eps=1e-8, weight_decay=0
                 state['exp_avg'] = p.data.new().resize_as_(p.data).zero_().share_memory_()
                 state['exp_avg_sq'] = p.data.new().resize_as_(p.data).zero_().share_memory_()
                 
-        def step(self, closure=None):
-            for group in self.param_groups:
-                for p in group['params']:
-                    if p.grad is None: continue
-                    self.state[p]['shared_steps'] += 1
-                    self.state[p]['step'] = self.state[p]['shared_steps'][0] - 1 # a "step += 1"  comes later
-            super.step(closure)
```

Comment:
why did u override step()? if you look at [pytorch's](https://github.com/pytorch/pytorch/blob/master/torch/optim/adam.py#L56) default implementation, it doesn't seem to conflict with the general A3C model.